mlflow/server/js/src/components/RunsTableColumnSelectionDropdown.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Button, Dropdown, Icon, Menu } from 'antd';
+import { SearchTree } from '../common/components/SearchTree';
+import { ColumnTypes } from '../common/utils';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import ExperimentViewUtil from './ExperimentViewUtil';
+
+export class RunsTableColumnSelectionDropdown extends React.Component {
+  static propTypes = {
+    paramKeyList: PropTypes.array,
+    metricKeyList: PropTypes.array,
+    visibleTagKeyList: PropTypes.array,
+    onCheck: PropTypes.func,
+    categorizedUncheckedKeys: PropTypes.object.isRequired,
+  };
+
+  static defaultProps = {
+    paramKeyList: [],
+    metricKeyList: [],
+    visibleTagKeyList: [],
+  };
+
+  state = {
+    menuVisible: false,
+  };
+
+  handleCheck = (checkedKeys, allKeys) => {
+    console.log('checkedKeys', checkedKeys);
```

Comment:
This line should be removed.

Reply:
Removed

Reply:
Maybe we should let the linter detect `console.log`.

Reply:
I think it's OK to leave it open for now. It's used a lot during development phase, we probably don't want running a lint fix to auto remove them.

Reply:
Agreed.